ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcarousel/v1/productcarousel/productcard.html

```diff
@@ -14,9 +14,13 @@
 <template data-sly-template.productcard="${@ product}">
     <div role="displaycard" class="card product__card">
         <a href="${product.URL}">
-            <img class="product__image" src="${product.imageURL}" data-gallery-role="currentimage" />
-            <div>${product.Title}</div>
-            <div class="product__price">${product.Currency} ${product.Price}</div>
+            <div class="product__card--image">
```

Comment:
I don't think this is valid here, the `--image` suffix is a BEM modifier that should define a "behavior" or "state", not a child element.

Reply:
In this case, it should be `product__card__image`